### ISSUE TITLE
CommentReplies is always empty

### ISSUE THREAD

**Issue by ShinyRa** [SuspiciousLookingOwl/youtubei#114]
**Bug description**
Fetching replies of a comment using the ```.next()``` function on always returns an empty array.

**To Reproduce**
```javascript
import { Client } from "youtubei";

const youtube = new Client();

const run = async () => {
    const video = await youtube.getVideo("xA-BxSKV0aU")
    const comments = await video.comments.next()
    console.log(comments[0].replyCount) // 49, indicating that 49 replies could be fetched
    
    const replies = await comments[0].replies.next()
    console.log(replies) // empty array
}

run()
```
**Output**
![image](https://github.com/user-attachments/assets/90eaf9d4-0588-4678-9aeb-3b1970b31ea7)

**Expected behavior**
When retrieving replies of a youtube comment using ```comment.replies.next()```, with ```comment.replyCount``` being higher than 0. I expect the replies array to be populated with correctly fetched replies.

**Comment by ShinyRa** [comment]
After some minor debugging, I found that an instance of ```CommentReplies``` gets created while getting comments, but calling ```const replies = await comments[0].replies.next()``` does not trigger the ```fetch()``` method in ```CommentReplies```.

By manually calling the fetch method using ```const replies = await comments[0].replies.fetch()```, I get an error (see attached screenshot), but granted this is not the intended way of retrieving replies.

Maybe I'm looking in the wrong places and retrieving comment replies is done differently, if that is the case please let me know!

![image](https://github.com/user-attachments/assets/25dd8de3-3de8-4576-bdd3-9199071c04aa)

**Comment by SuspiciousLookingOwl** [comment]
Fixed on `1.6.3`
